settings.php

```diff
@@ -35,6 +35,9 @@
 
 if (!$ADMIN->locate('tool_objectfs')) {
```

Comment:
Do you understand why this if exists? This seems very non standard and I suspect might be the root cause of the weirdness